version to 0.10.0

scottdraves opened the pull request.

scottdraves committed:
version to 0.8.0, add maven dep

scottdraves committed:
Merge pull request #1 from twosigma/spot/0.8.0

version to 0.8.0, add maven dep

scottdraves committed:
0.8.1

scottdraves committed:
Merge branch 'master' into master

scottdraves committed:
Merge branch 'master' into master

scottdraves committed:
merge

scottdraves committed:
0.9.1

jmsdnns committed:
version bump

scottdraves committed:
Merge pull request #2 from jmsdnns/master

version bump

scottdraves committed:
merge

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
@conda-forge-admin, please rerender.

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.4

scottdraves committed:
revert weird looking commit:
Author: conda-forge-admin <pelson.pub+conda-forge@gmail.com>
Date:   Mon Dec 18 23:47:02 2017 +0000

    MNT: Re-rendered with conda-smithy 2.4.4

jakirkham commented:
So the commit added is a re-rendering. It cut the Windows 32-bit builds and Python 2 builds because of [these `skip` lines]( https://github.com/conda-forge/beakerx-feedstock/blob/d7c99227e800d31db17e9c5a5a6afd6465ed522f/recipe/meta.yaml#L20-L21 ). While the builds can be manually added back to the CIs, they are not producing packages currently and have not been for some time, which is a suboptimal use of the orgs resources. If those are meant to be built, would recommend dropping the `skip` lines and then re-rendering. Otherwise it would be good not to run builds that do not produce packages.

scottdraves commented:
thanks. the changes seemed ok but i didn't see that you triggered them, and they happened at the same time as our PR,  because i was looking at "git log" not here, and i didn't see them on https://github.com/conda-forge/beakerx-feedstock/commits/master, sorry about that.  will put them back.

scottdraves committed:
undo undo

jakirkham commented:
No worries. This is part of a [new webservice]( https://conda-forge.org/docs/webservice.html ) that we have been trying out of late, which you are welcome to use as well. If you have any suggestions on how to make things more clear, happy to integrate them into the procedure.

jakirkham commented:
On a different note, it seems like you are running into [an issue that I encountered earlier today]( https://github.com/conda-forge/jupyter_client-feedstock/pull/14#issuecomment-352510987 ) (e.g. [this build]( https://ci.appveyor.com/project/conda-forge/beakerx-feedstock/build/1.0.42/job/t78a32sqdr2xoh7l#L12991 )). Might try installing with `pip` instead to see if that resolves the issue.

jmsdnns commented:
Interesting. Your issue looks exactly the same. Will look into that. Thanks!

jmsdnns committed:
Addressing issue with conda-forge

scottdraves committed:
Merge pull request #3 from jmsdnns/master

Addressing issue with conda-forge

jakirkham commented on recipe/meta.yaml:
```diff
@@ -16,10 +16,10 @@ source:
   skip: true  # [py2k]
   skip: true  # [win32]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install .
```
Will need the `--no-deps` argument as well.

jakirkham commented on recipe/meta.yaml:
```diff
@@ -16,10 +16,10 @@ source:
   
 build:
-  number: 0
+  number: 1
```
This can stay `0` as we are doing a new release and there are no broken builds that we are fixing yet.

jmsdnns committed:
Another fix to work with conda-forge

jmsdnns committed:
Merge pull request #4 from jmsdnns/master

Another fix to work with conda-forge

jmsdnns committed:
Adding -e flag

jmsdnns committed:
Merge pull request #5 from jmsdnns/master

Adding -e flag

jmsdnns committed:
Finally worked out that conda-forge is running pytest by depending on us to list it as a dependency. The error message that said a usable python version was not found is incorrect for what the error actually was, which is that pytest wasn't found.

jmsdnns committed:
Merge pull request #6 from jmsdnns/master

Adding pytest as dependency.

jmsdnns committed:
Reverting back to the original command for installing

jmsdnns committed:
Merge pull request #7 from jmsdnns/master

Reverting back to the original command for installing

jmsdnns committed:
Still seeing errors related to pytest, so adding it as dependency for the run step too

jmsdnns committed:
Merge pull request #8 from jmsdnns/master

Still seeing errors related to pytest, so adding it as dependency for…
